### DISCUSSION TITLE
Сhange window size and lock position

### DISCUSSION THREAD

**Discussion by Sribnyi-Ruslan** [IBM/mac-ibm-notifications#151] [Q&A] [answered]
Hello! Is there any possibility to specify the size of the progressbar window, and fix the IBM Notifier window, for example, in the center of the screen, without the possibility of dragging and collapsing it?

**Comment by SMartorelli** [comment, marked as answer]
Hi @Sribnyi-Ruslan the size of the IBM Notifier UIs are dynamically set based also on the accessibility settings an user had set on his Mac so we don't offer the chance to set a different size to avoid issue with the representation of the UI elements. 
The popup UI is not collapsable and the onboarding one provide the option to hide the title bar buttons to avoid it to be easily closed/collapsed. 
While being technically possible on macOS, we don't provide the option to make it unmovable as it could lead to a poor user experience. 
IBM Notifier provide anyway the option to keep the UI always on top of the view hierarchy on screen with the `-always_on_top` argument.
If you think that would be great for your workflows to have an unmovable UI feel free to open a [Feature Request](https://github.com/IBM/mac-ibm-notifications/issues/new?assignees=&labels=&template=feature_request.md&title=%5BFeature+Request%5D) and we'll work on the implementation of it in the next releases.

**Comment by Sribnyi-Ruslan** [comment]
Understand. 
Thank you very much for your detailed answer and quick response. I want to express my thanks for such a cool product, I used jamf helper and apple script before and it was a pain)

**Comment by SMartorelli** [comment]
Anytime! Feel free to ask if you have doubts. You can also join the `#mac-ibm-open-source` Slack channel in the [MacAdmins](https://join.slack.com/t/macadmins/shared_invite/zt-1l1rzyc9j-~6Cm75OJoSChZUoz_eJ6nQ) workspace to have faster interaction :wave:
